src/components/VtkTwoView.vue

```diff
@@ -563,7 +579,7 @@ export default defineComponent({
     // --- apply windowing and slice configs --- //
 
     watchEffect(() => {
-      if (sliceConfig.value === null || wlConfig.value === null) {
+      if (!sliceConfig.value || !wlConfig.value) {
```

Comment:
Isn't it better to explicit the `null` ness?

Reply:
This can be reverted back to explicit null checks. Thanks for catching that.